packages/shared-ux/chrome/navigation/src/ui/default_navigation.tsx

```diff
@@ -194,6 +194,7 @@ export const DefaultNavigation: FC<ProjectNavigationDefinition & Props> = ({
         }
 
         if (isGroupDefinition(navNode)) {
+          if (navNode.sideNavStatus === 'hidden') return null;
```

Comment:
This should not be here. Remember that all the definition prop can also be added to the components, so `Navigation.Group` in this case.

You need to add this `if` statement here:
https://github.com/elastic/kibana/blob/main/packages/shared-ux/chrome/navigation/src/ui/components/navigation_group.tsx#L84

Reply:
thanks Seb! I moved the condition